### DISCUSSION TITLE
Is Rancher Desktop using the docker community version or moby?

### DISCUSSION THREAD

**Discussion by Luxvao** [rancher-sandbox/rancher-desktop#3214] [Q&A] [answered]
If I decide to use the docker cli with rancher desktop, would I be using the community version of docker and would I have to log in? EDIT: Whats the difference between using docker desktop and rancher desktop with docker cli?
Thanks

**Comment by jandubois** [comment, marked as answer]
> If I decide to use the docker cli with rancher desktop, would I be using the community version of docker and would I have to log in?

Rancher Desktop only uses open source components, so moby and the docker-cli from the community repos.

You don't have to log into Dockerhub to access public images, but you obviously will have to authenticate to access private images or to push images back to the hub. You'll also get higher bandwidth limits when you log in, even with a free/personal account.

> Whats the difference between using docker desktop and rancher desktop with docker cli?

The basic functionality is very similar, but Docker Desktop is a proprietary implementation for things like network and volume management, so will work differently. You have to try them to see if the differences matter to you.

Also Docker Desktop requires a paid account if you use it in a company over a certain size.

**Reply by Luxvao** [reply]
Okay, thank you.
